base/linalg/cholesky.jl

```diff
@@ -78,83 +104,149 @@ function chol!{T}(A::AbstractMatrix{T}, ::Type{LowerTriangular})
     return LowerTriangular(A)
 end
 
-"""
-    chol(A::AbstractMatrix) -> U
-
-Compute the Cholesky factorization of a positive definite matrix `A`
-and return the UpperTriangular matrix `U` such that `A = U'U`.
-"""
-function chol{T}(A::AbstractMatrix{T})
```

Comment:
if I'm reading this right, is there no longer a definition for AbstractMatrix at all?